docs/ABOUT.md

```diff
@@ -0,0 +1,14 @@
+Javascript is a scripting language used to provide dynamic and interactive content on webpages. It does not have class-oriented inheritance, but it does have prototype-oriented inheritance. JavaScript provides benefits like :- 
+* JavaScript in the browser can reduce server traffic by validating user input before it is sent to the server.
```

Comment:
Your preceding sentence says you're going to list benefits.  I would begin each of the benefit bullet points with a verb:

1) Reducing server traffic by....
2) Providing immediate feedback...
(also "page" should probably be "pages" in the above bullet point)
3) Allowing richer user interfaces with content changes on ...